osd/rosa_clusterlogging_general.json

```diff
@@ -0,0 +1,9 @@
+{
+ "severity": "Error",
+ "service_name": "SREManualAction",
+ "cluster_uuid": "${CLUSTER_UUID}",
+ "summary": "Action required: Update Cluster Logging configuration",
+ "description": "Your cluster requires you to take action. Your cluster has problems on the Cluster Logging service which includes but is not limited to cluster logging operator, fluentd. Please refer to the following documentation for detailed instructions: https://docs.openshift.com/rosa/logging/rosa-install-logging.html",
```

Comment:
As a customer I wouldn't have much confidence that I could find and resolve this issue. cluster-logging-operator, fluentd and possibly other components? I believe fluentd is part of cluster-logging-operator. Can we be more specific? Would going through the docs "Verification steps" be the best place to start?